crates/hyperqueue/src/dashboard/ui/screens/home/chart_section.rs

```diff
@@ -0,0 +1,132 @@
+use std::time::{Duration, SystemTime, UNIX_EPOCH};
+use tako::messages::gateway::CollectedOverview;
+
+use crate::dashboard::ui::terminal::DashboardFrame;
+use tui::layout::{Alignment, Rect};
+use tui::style::{Color, Modifier, Style};
+use tui::symbols;
+use tui::text::{Span, Spans};
+use tui::widgets::{Axis, Block, Borders, Chart, Dataset, Paragraph, Wrap};
+
+/// A new WorkerTimeState is created everytime there is a #worker change event
+#[derive(Clone)]
+struct WorkerCountAtTime {
```

Comment:
AtTime is a bit unwieldy. Let's use some unified name for items that can be drawn on a chart. Also remove `pub`.
For example:

```rust
struct WorkerCountRecord {
    time: SystemTime,
    worker_count: u32
}
```